macros/utils/run_queries/run_query.sql

```diff
@@ -0,0 +1,8 @@
+{% macro run_query(query) %}
```

Comment:
the name here is a bit dangerous (I know we should use the "elementary"), I'd just create a macro called "lower_column_in_query_result" and call it on the result that gets back from the run_query macro

Reply:
This is needed on nearly every query we're doing in order to support Snowflake,
so I think it's useful to have a macro that abstracts that.
I can rename the macro to convey that it's doing it, but not sure if it's a must.
(Our current way of dealing with it is by using `insensitive_get_dict_value` everywhere.)

What do you think?